test/libyul/yulOptimizerTests/fullSuite/switch_inline_no_match.yul

```diff
@@ -0,0 +1,21 @@
+{
+	mstore(f(3), 0)
+	function f(x) -> y {
+		switch x
```

Comment:
Actually in this case, we should just remove the `switch` altogether, shouldn't we?

Reply:
And could you please copy these tests into `yulOptimizer/structuralSimplifier` (of couse without the function in that case)?

Reply:
and what do we put there instead of the `f(3)`?

Reply:
just use
```
let x := 3
switch x
...
```
without any functions.

Reply:
I am just confused how the compiler handles a switch with no match. Is it an error?

Reply:
no, it just skips the switch

Reply:
okay, so I remove the whole switch if no mach. Got it.